SU-SWS/stanford_profile
# DEC-152: User roles and permissions

# READY FOR REVIEW

# Summary
- Adds new default users to match the new default roles

# Review By (Date)
- EOD Tuesday October 15

# Urgency
- Medium, we'd like to get some of this out to the pilot.

# Steps to Test

1. Read and review https://docs.google.com/document/d/1X3lqcf7m1awvF6J8PkxdfUy-ZAiBklx9GUKMf2LOvxA/edit
2. Check out this branch
3. Run full site install
4. Log in as an administrator
5. Validate users and roles are there with no typos
6. Review permissions page and compare options to the google document
7. Run tests

# Affected Projects or Products
- This profile

# Associated Issues and/or People
- https://docs.google.com/document/d/1X3lqcf7m1awvF6J8PkxdfUy-ZAiBklx9GUKMf2LOvxA/edit
- D8CORE-152
- https://github.com/SU-SWS/acsf-cardinalsites/pull/27

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
